### PR TITLE
raidboss: Simple Aloalo triggers

### PR DESCRIPTION
**Pull request by kshman**
Sorry no timeline and no i18n. 😭
Simple triggers only.

### CODE REVIEW

File: ui/raidboss/data/06-ew/dungeon/aloalo.ts [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+        second: {
+          en: 'Under Orbs',
+          ja: 'ドーナツ、玉の下へ',
+        },
+        third: {
+          en: 'Under Donut Orbs',
+          ja: 'ドーナツ玉の下へ',
+        },
+        text: {
+          en: 'Becareful Orbs',
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/aloalo.ts:72]:
```suggestion
          en: 'Under Donut Far From Axe',
```

File: ui/raidboss/data/06-ew/dungeon/aloalo.ts [hunk 2]
```diff
@@ -0,0 +1,688 @@
...
+      },
+    },
+    {
+      id: 'Aloalo Quaqua Arcane Armaments Trident',
+      type: 'StartsUsing',
+      netRegex: { id: '8B9F', source: 'Quaqua', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Outside of between tridents',
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/aloalo.ts:121]:
```suggestion
          en: 'Outside between tridents',
```

**Comment by quisquous** [comment]
Looks good, thank you!

**Comment by kshman** [comment]
Thank you!

File: ui/raidboss/data/06-ew/dungeon/aloalo.ts [hunk 3]
```diff
@@ -0,0 +1,688 @@
...
+        text: {
+          en: 'Look Away from Rings',
+          de: 'Schau von den Ringen weg',
+          fr: 'Ne regardez pas l\'anneau',
+          ja: '輪から視線回避',
+          cn: '视线避开圆环',
+          ko: '고리 모양 눈 시선 피하기',
+        },
+      },
+    },
```
[the remark this answers is not in the document]
Comment (quisquous) [reply ui/raidboss/data/06-ew/dungeon/aloalo.ts:160]: Oh sorry, this must be the right path that I haven't done yet.

**Pull request merged**